scvi/inference/posterior.py

```diff
@@ -376,8 +436,10 @@ def update_sampler_indices(self, idx: Union[List, np.ndarray]):
         >>> # Do not forget next line!
```

Comment:
is this comment still needed? Can't see on the review

Reply:
yes, this is a weird hacky legacy thing we had for changing indices temporarily to a cell type in DE